test/sim/abilities/neutralizinggas.js

```diff
@@ -201,7 +201,7 @@ describe('Neutralizing Gas', function () {
 
 		// Gluttony now has the opportunity to activate the Aguav Berry again on taking damage
 		battle.makeChoices();
-		assert.equal(wynaut.hp, Math.floor(wynaut.maxhp / 2) - 1 + Math.floor(wynaut.maxhp * 0.33));
+		assert.equal(wynaut.hp, Math.floor(wynaut.maxhp / 2) - 1 + Math.floor(wynaut.maxhp / 3));
```

Comment:
I did have to change the test, was getting off by one on the berry healing

Reply:
Looks like this got missed when https://github.com/smogon/pokemon-showdown/commit/05faddbc93f449e6f494e60c5a884dd31a7803fa landed, good catch